examples/games/desktoy.rs

```diff
@@ -0,0 +1,320 @@
+//! Bevy logo as a desk toy! Now with Googly Eyes!
+
+use bevy::{
+    app::AppExit,
+    input::common_conditions::{input_just_pressed, input_just_released},
+    prelude::*,
+    sprite::{MaterialMesh2dBundle, Mesh2dHandle},
+    window::{PrimaryWindow, WindowLevel},
+};
+
+#[cfg(target_os = "macos")]
+use bevy::window::CompositeAlphaMode;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.set(WindowPlugin {
+            primary_window: Some(Window {
+                title: "Bevy Desk Toy".into(),
+                transparent: true,
+                #[cfg(target_os = "macos")]
+                composite_alpha_mode: CompositeAlphaMode::PostMultiplied,
+                ..default()
+            }),
+            ..default()
+        }))
+        .insert_resource(ClearColor(Color::srgb(0.2, 0.2, 0.2)))
+        .insert_resource(CursorWorldPos(None))
+        .add_systems(Startup, setup)
+        .add_systems(
+            Update,
+            (get_cursor_world_pos, update_cursor_hit_test).chain(),
+        )
+        .add_systems(
+            Update,
+            (
+                start_drag.run_if(input_just_pressed(MouseButton::Left)),
+                end_drag.run_if(input_just_released(MouseButton::Left)),
+                drag.run_if(resource_exists::<DragOffset>),
+                quit.run_if(input_just_pressed(MouseButton::Right)),
+                enable_transparency.run_if(input_just_pressed(KeyCode::Space)),
```

Comment:
Should this also have a `run_once` run condition?

Reply:
I changed this to be toggle-able.